### PR TITLE
Update readme docs for migrations

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | no
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIG-262

### CODE REVIEW

File: ecms_base/modules/custom/ecms_migration/README.md [hunk 1]
```diff
@@ -12,6 +12,21 @@ extension will crawl the site and find all publicly accessible URLs.
...
+### Files list and URL encoding
+During early testing some issues with handling URL encoded file names were
+encountered. To address this, the files lists should be cleared of all "%20"
+encoded spaces, and simply replaced with a space. The resulting file and file
+redirect imports will encode the path automatically.
+
+A more reliable tool is the [Screaming Frog] app. This has been used on the
+last few migrations. In addition to the crawl of a site, it can also
+be used in "[List Mode]" mode to test the resulting migration. Simply switch to
+list mode, the paste in all the URLs you want to test. To generate the list,
```
Comment (JMStewart00) [inline ecms_base/modules/custom/ecms_migration/README.md:24]: Typo here: list mode, **then** paste in all the URLs you want to test.

**Review by JMStewart00 — CHANGES_REQUESTED** [review]
One nitpick typo in there then it should be good to go.

File: ecms_base/modules/custom/ecms_migration/README.md (hunk 1 above)
Comment (ben-hamelin) [reply ecms_base/modules/custom/ecms_migration/README.md]: Check on that! Thanks!

**Review by JMStewart00 — APPROVED** [review]
Looking good! /approved

**Pull request merged**